### PR TITLE
feat(Desu-Online): fix for episode title

### PR DESCRIPTION
**Pull request by Mefuuu**
## Description 
* If the episode title had a `-` character, it only displayed the first title segment before the `-` character in the status. This commit fixes that

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

obraz_2022-10-17_183647391

</details>

### CODE REVIEW

File: websites/D/Desu-Online/presence.ts [hunk 1]
```diff
@@ -127,8 +127,14 @@ presence.on("UpdateData", async () => {
...
-		if (playinfo.length > 2)
-			presenceData.state += ` - "${playinfo[1].slice(1, -1)}"`;
+		if (playinfo.length > 2) {
+			let episodetitle = "";
+			for (let i = 0; i < playinfo.length; i++) {
+				if (i !== 0 && i !== playinfo.length - 1)
+					episodetitle += `${playinfo[i]}-`;
+			}
+			presenceData.state += ` - "${episodetitle.slice(1, -2)}"`;
+		}
```
Comment (DarkVillager) [inline websites/D/Desu-Online/presence.ts:137]: Or just do this:

`.split('- desu-online.pl')[0]`

**Review by DarkVillager — CHANGES_REQUESTED** [review]

File: websites/D/Desu-Online/presence.ts (hunk 1 above)
Comment (DarkVillager) [inline websites/D/Desu-Online/presence.ts:137]: Couldn't you just do this:
`document.querySelector('[class="selected"]').querySelector("span").textContent.split(/- [0-9]/gm)[0]`

**Review by DarkVillager — CHANGES_REQUESTED** [review]

File: websites/D/Desu-Online/presence.ts (hunk 1 above)
Comment (Mefuuu) [reply websites/D/Desu-Online/presence.ts:137]: i could but idk how regex works ;v
anyway ty

Comment (Mefuuu) [reply websites/D/Desu-Online/presence.ts:137]: i want text without `Odc 1 -`, only `Pies i piła mechaniczna`

Comment (Mefuuu) [reply websites/D/Desu-Online/presence.ts:137]: image

Comment (DarkVillager) [reply websites/D/Desu-Online/presence.ts:137]: then just change my suggested regex

Comment (Mefuuu) [reply websites/D/Desu-Online/presence.ts:137]: the thing is i don't really know how regex works

Comment (Mefuuu) [reply websites/D/Desu-Online/presence.ts:137]: nvm i found solution

**Review by EncryptedDev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**